aws-distro-opentelemetry-node-autoinstrumentation/src/aws-opentelemetry-configurator.ts

```diff
@@ -190,6 +206,32 @@ export class AwsOpentelemetryConfigurator {
     return autoResource;
   }
 
+  private getPropagator(): TextMapPropagator {
+    if (process.env.OTEL_PROPAGATORS == null || process.env.OTEL_PROPAGATORS.trim() === '') {
+      return new CompositePropagator({
```

Comment:
is there a better option to not rewrite this function by copying it from upstream? can we just patch `propagatorMap` for `auto-configuration-propagators/utils` file?

Reply:
See #171, we revert to using the upstream dependency